packages/contracts/.env.example

```diff
@@ -5,4 +5,7 @@ RPC_URL=
 PRIVATE_KEY=
 
 # Used to verify contracts on etherscan. This is optional
-ETHERSCAN_API_KEY=
\ No newline at end of file
+ETHERSCAN_API_KEY=
+
+# Contract owner, used to deploy Diamond Contract
+OWNER=
```

Comment:
Why do we need OWNER env here? We have `PRIVATE_KEY` value which will be used to deploy contracts even Diamond contract. If you have any specific reason to use it, please keep me updated.

Reply:
Last I recall the environment variable key is UBQ or something like that

Reply:
> Last I recall the environment variable key is `UBQ_DEPLOYER` or something like that

Exactly, that's what I intended